pkg/cvo/metrics.go

```diff
@@ -196,12 +196,22 @@ func (m *operatorMetrics) Collect(ch chan<- prometheus.Metric) {
 				ch <- g
 			}
 		}
-		if failing != nil && failing.Status == configv1.ConditionTrue {
-			g := m.version.WithLabelValues("failure", current.Version, current.Image, completed.Version)
-			if failing.LastTransitionTime.IsZero() {
-				g.Set(0)
+		if failing != nil {
+			if failing.Status == configv1.ConditionTrue {
+				g := m.version.WithLabelValues("failure", current.Version, current.Image, completed.Version)
+				if failing.LastTransitionTime.IsZero() {
+					g.Set(0)
+				} else {
+					g.Set(float64(failing.LastTransitionTime.Unix()))
+				}
+				ch <- g
+			}
+
+			g := m.clusterOperatorConditions.WithLabelValues("cluster-version", "Failing", string(failing.Reason))
```

Comment:
the name should be `version`. all the clusteroperators are already cluster.. it is redundant. Also the ClusterVersion object is also named version.

Reply:
Fixed in b69cd48 -> d881086.